docs/elements.md

```diff
@@ -347,6 +350,76 @@ Attribute | Type | Default | Description
 
 -----
 
+### `pl-order-blocks` element
+
+Element to arrange given blocks of code or text that are displayed initially in the *input area*. The blocks can be moved to the *output area* to construct the solution of the problem.  In the example below, the input area is denoted by the header "Drag from here" and the output area is denoted with the header "Construct your solution here".
+
+#### Sample element
+
+![](elements/pl-order-blocks.png)
+
+**question.html**
+
+```html
+<pl-question-panel>
+<p> How can we compute $c$, the summation of two given numbers $a$ and $b$?</p>
+</pl-question-panel>
+<pl-order-blocks answers-name="sum_numbers" >
+  <pl-answer correct="false">$a * b$</pl-answer>
+  <pl-answer correct="true">$a + b$</pl-answer>
+  <pl-answer correct="false">$a + c$</pl-answer>
+  <pl-answer correct="false">$b + c$</pl-answer>
+</pl-order-blocks>
+```
+
+
+#### Customizations
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`answers-name` | string | — | Variable name to store data in.
+`weight` | integer | 1 | Weight to use when computing a weighted average score over all elements in a question.
+`shuffle-options` | boolean | false | Specify whether the ordering of the blocks in the input area should be shuffled. If false, the ordering of the answers in the HTML file (defined by `<pl-answer>`) will be used.
+`max-incorrect` | integer | special | The maximum number of incorrect answers to be displayed in the input area. The incorrect answers are set using `<pl-answer correct="false">`. Defaults to displaying all incorrect answers.
+`min-incorrect` | integer | special | The minimum number of incorrect answers to be displayed in the input area. The incorrect answers are set using `<pl-answer correct="false">`. Defaults to displaying all incorrect answers.
```

Comment:
Why don't we have `max/min-correct` as well?

Reply:
originally we only had `max-incorrect` and `min-incorrect` from one of the attributes in `pl-parsons-problem`: `pl-distractor`. `pl-parsons` doesn't have a `max/min-correct` so it wasn't implemented. Should we look into adding that then?

Reply:
@mwest1066 We are starting with `max-incorrect` and `min-incorrect` for compatibility with questions from CS 105. I say we merge the code as it is now, and later we can add other attributes.

Reply:
I'm fine to merge this PR now and add max/min-correct later

Reply:
Added this TODO to #3476